packages/lib/modules/lbp/steps/sale-structure/ProjectedPriceChart.tsx

```diff
@@ -27,6 +42,8 @@ export function ProjectedPriceChart({ startDate, endDate, onPriceChange, prices,
 
   const chartInfo: EChartsOption = {
```

Comment:
<img width="620" alt="Screenshot 2025-07-07 at 08 53 59" src="https://github.com/user-attachments/assets/9ac07ca3-f99e-42bb-8e26-a99f4a2e6dff" />

Some of the changes around here have made the price labels to be cut on the creation page.

Reply:
fixed this